src/main/java/io/phasetwo/keycloak/magic/auth/token/MagicLinkActionToken.java

```diff
@@ -25,6 +25,8 @@ public class MagicLinkActionToken extends DefaultActionToken {
   @JsonProperty(value = JSON_FIELD_REMEMBER_ME)
   private Boolean rememberMe = false;
 
+  private Boolean actionTokenPersistent = true;
```

Comment:
This needs a @JsonProperty that has a value that is small (e.g. "ru" for reusable) to keep the token size as small as possible.